Layout/UITableView+Layout.swift

```diff
@@ -428,8 +423,7 @@ extension UITableViewHeaderFooterView {
             let idExpression = LayoutExpression(expression: expression, type: RuntimeType(String.self), for: node)
             reuseIdentifier = try idExpression.evaluate() as? String
         }
-        let view = self.init() // Workaround for `self.init(reuseIdentifier:)` causing build failure
-        view.setValue(reuseIdentifier, forKey: "reuseIdentifier")
+        let view = self.init(reuseIdentifier: reuseIdentifier)
```

Comment:
This is the line that crashes the Xcode 8.3.3 compiler when optimizations are enabled.